modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,12 @@
     <string name="empty" translatable="false" />
 
     <string name="account">Account</string>
+    <string name="add_bookmark">Add bookmark</string>
+    <string name="add_bookmark_title">Add Bookmark</string>
```

Comment:
Can't we use `add_bookmark` string for the title?  Other titles like "Create folder", "Select podcasts" too do not capitalize the first letter of the second word.